pylinalg/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 # flake8: noqa
 
-__version__ = "0.3.3"
+__version__ = "0.3.4"
```

Comment:
```suggestion
import pkg_resources

__version__ = pkg_resources.get_distribution('pylinalg').version
```

Shall we can reduce the places where we have to bump version numbers by 1?

Reply:
Sure, but then we should use importlib as it is replacing pkg_resources. Let's do that in a future PR.